src/manager/MapFishPrintV3Manager.js

```diff
@@ -71,6 +71,13 @@ export class MapFishPrintV3Manager extends BaseMapFishPrintManager {
    */
   _printJobReference = null;
 
+  /**
+   * Default timeout in ms after which print job polling will be canceled.
+   *
+   * @type {Number}
+   */
+  timeout = 5000;
```

Comment:
Please use underscores for private variables here as well.

Reply:
It's not a private but a config which can be provided during manager initialization like 

```
const printManager = new MapFishPrintV3Manager({
      url: 'http://someurl.com',
      [...]
      timeout: 30000
      [...]
    });
```
And configs don't have any underscores (compare [`BasMapFishPrintManager`](https://github.com/terrestris/mapfish-print-manager/blob/master/src/manager/BaseMapFishPrintManager.js#L65) class for instance)